### PR TITLE
Add user guide for cli wallet daemon isolation using Qubes and Whonix.

### PR DESCRIPTION
**Pull request by 0xB44EFD8751077F97**
Finally got around to perfecting this guide, sorry it took so long. IMO this is the safest wallet outside from a hardware wallet. Let me know what you all think.

### CODE REVIEW

**Comment by mattcode55** [comment]
At a glance it all looks good, I'll try following the guide later :)

**Comment by 0xB44EFD8751077F97** [comment]
Should the guide use a `systemd` unit or not? I've come to the opinion that either the `monerod.service` should have a `Restart=` configuration or the guide should switch out the .service for just putting the startup command in `/rw/config/rc.local`. 

I've made a PR adding a `Restart=always` to the `monerod.service` file: https://github.com/monero-project/monero/pull/2831.

**Comment by 0xB44EFD8751077F97** [comment]
Since https://github.com/monero-project/monero/pull/2831 got merged I added the changes to the guide and rebased the commits. Guide now uses a `systemd` file with `Restart=always` and `PrivateTmp=true`.

**Comment by luigi1111** [comment]
@mattcode55 any update? Would like a reviewer or two to approve.

**Comment by mattcode55** [comment]
@luigi1111 The guide looks good to me, but I haven't really looked at Qubes too much so I don't feel too confident approving it.

**Comment by rehrar** [comment]
@0xB44EFD8751077F97 @luigi1111 @mattcode55 Works like a charm. Qubes 4.0-rc4. Syncing right now. Will report back very soon.

@0xB44EFD8751077F97 please rebase so this can be merged.

**Comment by 0xB44EFD8751077F97** [comment]
Well, I tried rebasing and fixing for the changes to the site. It's been a long day, and 3 months of merges to deal with. All on top of the site structure changing. Let me know if there's a problem with the git history or anything.

On another note, contributing guides to the site has gone from difficult to a complete turnoff of the past few months. I don't doubt there are contributors being turned away by the complexity to the steps.

File: _i18n/en/resources/user-guides/cli_wallet_daemon_isolation_qubes_whonix.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
+# CLI Wallet/Daemon Isolation with Qubes + Whonix
+
+With [Qubes](https://qubes-os.org) + [Whonix](https://whonix.org) you can have a Monero wallet that is without networking and running on a virtually isolated system from the Monero daemon which has all of its traffic forced over [Tor](https://torproject.org).
+
+Qubes gives the flexibility to easily create separate VMs for different purposes. First you will create a Whonix workstation for the wallet with no networking. Next, another Whonix workstation for the daemon which will use your Whonix gateway as it's NetVM. For communication between the wallet and daemon you can make use of Qubes [qrexec](https://www.qubes-os.org/doc/qrexec3/).
```
Comment (leonklingele) [inline _i18n/en/resources/user-guides/cli_wallet_daemon_isolation_qubes_whonix.md]: `its NetVM` (no apostrophe)

**Pull request merged**